freqtrade/freqai/utils.py

```diff
@@ -85,6 +93,83 @@ def get_required_data_timerange(
     return data_load_timerange
 
 
+def auto_populate_any_indicators(
```

Comment:
i'm a bit concerned about this - we should call this functions from tests to ensure it's working (and keeps working).
we assume people trust this with just the "flick of a switch" - so we should be sure it'll not break at random ...

Reply:
Yes, this needs a test for sure.